standard.md

```diff
@@ -15,9 +15,8 @@ Indicator |	Requirement
 **7. Adherence to privacy and applicable laws** |	The project must state that it complies with relevant privacy laws, and all applicable international and domestic laws.
 **8. Adherence to standards & best practices** |	Projects must demonstrate some adherence to standards, best practices and/or principles. i.e. the principles for digital development
 **9. Do No Harm** | 	All projects must demonstrate that they have taken steps to ensure that the project anticipates, prevents and does no harm. 
-**9a) Privacy & Freedom of Expression** |	All projects must have strategies in place to anticipate, respond to and minimize adverse impacts on privacy and freedom of expression where governments are believed to be using the project’s product or services for illegitimate or political purposes.
```

Comment:
There are technical approaches to the development of digital public goods that advance this purpose more than others. Perhaps requiring a statement about how the project accomplishes this goal, and to what extent, would be useful for transparency. 

I agree that how digital public goods are used is not easily policed.

Reply:
@sgoggins: I would like to ask you specifically what is your take on whether `strategies in place to anticipate, respond to and minimize adverse impacts on privacy and freedom of expression` is intrinsic to the development of digital public goods, or is an external factor tied to the implementation of that good out into the world. If it's the latter, it becomes a matter of policing which I would argue is out of the scope of this standard. On the other hand, if it is intrinsic, it should somehow be part of the standard.

What I am understanding from your `statement about how the project accomplishes this goal, and to what extent, would be useful for transparency` is something that seems to be under the purview of the the implementing party, and can be separated (completely?) from the actual digital public good.